Frontend/items.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE HTML>
+<html lang='en'>
+<head>
+    <title>Welcome to Lean Fridge</title>
+
+    <link rel='stylesheet' href='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css' integrity='sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7' crossorigin='anonymous'>
+
+<script type = 'text/javascript'>
+    var stock_array = [
+            {item_id: 1, Name: 'apple', count: '5', purchased_on: '2016-01-22', detail: 'organic Fuji apples'},
```

Comment:
follow a consistent pattern for the names in the json. (Name has n capital, nothing else does)
count should be an integer?